jmp-service-rest/src/main/java/com/gmail/iikaliada/controller/ServiceController.java

```diff
@@ -0,0 +1,93 @@
+package com.gmail.iikaliada.controller;
+
+import com.gmail.iikaliada.dto.SubscriptionRequestDto;
+import com.gmail.iikaliada.dto.SubscriptionResponseDto;
+import com.gmail.iikaliada.serivce.SubscriptionService;
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.Parameter;
+import io.swagger.v3.oas.annotations.media.Content;
+import io.swagger.v3.oas.annotations.media.Schema;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import io.swagger.v3.oas.annotations.responses.ApiResponses;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.context.request.WebRequest;
+
+import java.util.List;
+import java.util.NoSuchElementException;
+
+import static org.springframework.http.HttpStatus.NOT_FOUND;
+
+@RequestMapping("/api/subscriptionGroup")
+@RestController
+@RequiredArgsConstructor
+public class ServiceController {
+
+    private final SubscriptionService subscriptionService;
+
+    @Operation(summary = "Creates Subscription", parameters = {
+            @Parameter(name = "id", schema = @Schema(implementation = Long.class)),
+            @Parameter(name = "userId", schema = @Schema(implementation = Long.class))})
+    @ApiResponses(value = {
+            @ApiResponse(responseCode = "200", content = @Content(mediaType = "application/json",
+                    schema = @Schema(implementation = SubscriptionResponseDto.class))),
+            @ApiResponse(responseCode = "404", content = @Content)})
+    @RequestMapping(value = "/subscriptions", method = RequestMethod.POST)
```

Comment:
`@PostMapping` can be used. In this case, you don't need to specify **method** parameter